cmake/toolchains/o3d303.cmake

```diff
@@ -0,0 +1,22 @@
+#
+# Toolchain for cross-building imf3d for the O3D303 camera.
```

Comment:
May I recommend the usage of ``O3D3xx`` instead of ``O3D303``

Reply:
updated!